src/utils/scrappers/custom.ts

```diff
@@ -32,7 +32,7 @@ export async function customScrapper(options: ScrapperOptions) {
       elem = element?.querySelector(query);
     }
 
-    return elem ? elem.href : '';
+    return elem ? elem.href : (element ? element.href : '');
```

Comment:
why do you need this? if elem = element; 🤔

Reply:
Because it's being overridden by the previous `if`.  What I want to do is to return the self element's `href` in case the query selector fails to select an element. This happens because the current element is being selected on the `[class*="BaseListing_link"]` and there is no way to select the self element.